PlayCover/Model/PlayApp.swift

```diff
@@ -162,6 +167,9 @@ class PlayApp: BaseApp, ObservableObject {
         "com.pubg.newstate",
         "com.tencent.tmgp.pubgmhd",
         "com.dts.freefireth",
-        "com.dts.freefiremax"
-    ]
+        "com.dts.freefiremax",
+]
+    static let PROHIBITED_APPS_TIKTOK = [
```

Comment:
Riding on Depal's comment, this should be generalised to `MALICIOUS_APPS` so that the list can be expanded as new apps are discovered.

Reply:
Also, to keep consistency with `PROHIBITED_APPS`, it would be better to use the bundle identifier instead of the name of the app.

Reply:
thats optional o would say. i can do that for sure. in my opinion, bundle.name is better than bundle.identifier.